pymc3/tests/test_distributions.py

```diff
@@ -1363,36 +1378,47 @@ def test_binomial(self):
             Binomial,
             Nat,
             {"n": NatSmall, "p": Unit},
-            n_samples=10,
+            n_samples=-1,
```

Comment:
passed as expected

Reply:
You mean xfailed as expected?

Reply:
`test_binomial` was not expected to fail (not decorated with xfail)

Reply:
Nevermind I was confused by the collapsed diff